wiki/Legal/Copyright/Music_Licensing/en.md

```diff
@@ -0,0 +1,19 @@
+# Music Licensing
+
+The osu! team regularly reaches out to artists that the community loves and new artists whose music is appropriate for use within the community and game. Once an agreement is established, artists are listed under the [Featured Artists](/wiki/Featured_Artists) program.
+
+Artists interested in joining the Featured Artist program are encouraged to contact osu! for licensing negotiations.
+
+## Terms
+
+Music licensed by osu! is freely available for download from the [Featured Artists listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are beatmap templates prepared with timing data. A one minute automatically generated preview of each track is available for streaming on its artist listing. Because songs are used as a catalog for community members to create beatmaps, there is no guarantee that they will be used in the game.
```

Comment:
not sure if mentioning beatmaps/timing is a good idea for this page if it's meant to be read by ppl who may not know anything about osu

Reply:
I think it might have to be used in lamens terms since this could be read by either a record company or an artist who has not heard of the game before